src/script/src/python/rspython/vector_impl.rs

```diff
@@ -262,6 +285,38 @@ impl PyVector {
     }
 }
 
+impl Representable for PyVector {
+    #[inline]
+    fn repr_str(zelf: &Py<Self>, _vm: &VirtualMachine) -> PyResult<String> {
+        Ok(format!("{:#?}", *zelf))
+    }
+}
+
+impl AsNumber for PyVector {
+    fn as_number() -> &'static PyNumberMethods {
+        static AS_NUMBER: PyNumberMethods = PyNumberMethods {
```

Comment:
Again, don't use `to_owned` at all if you have other choices.

Reply:
> Again, don't use `to_owned` at all if you have other choices.

It seems the devs have attach some semantics to `to_owned`... it create a new Ref and inc ref cnt by one, and there seems not having a better way to do this convert other than a `to_owned`, maybe it's a wrong api design here?
```
impl ToOwned for PyObject {
    type Owned = PyObjectRef;

    #[inline(always)]
    fn to_owned(&self) -> Self::Owned {
        self.0.ref_count.inc();
        PyObjectRef {
            ptr: NonNull::from(self),
        }
    }
}
```

Reply:
Got it.  So there is no other choice, use `to_owned` is fine here.